internal/pkg/aws/s3/s3.go

```diff
@@ -155,6 +155,11 @@ func ParseURL(url string) (bucket string, key string, err error) {
 	return
 }
 
+// URL returns a virtual-hosted–style S3 url for the object stored at key in a bucket created in the specified region.
+func URL(region, bucket, key string) string {
```

Comment:
I guess this function is getting referenced in an upcoming PR is that correct 👀 ?